pkg/loadbalancer/loadbalancer.go

```diff
@@ -409,7 +409,7 @@ func (s *SVC) GetModel() *models.Service {
 		id  BackendID
 	}
 
-	if s == nil {
+	if s == nil || s.Type == SVCTypeNone {
```

Comment:
We should still return the model for service types `SVCTypeNone` so that we have that information visible with `cilium service list`